src/Core/Domain/RealTime/Model/Service.php

```diff
@@ -26,13 +26,16 @@
 use Core\Domain\RealTime\Model\Icon;
 use Core\Domain\RealTime\Model\Servicegroup;
 use Core\Domain\RealTime\Model\ServiceStatus;
+use Centreon\Domain\Common\Assertion\Assertion;
 
 class Service
 {
+    public const MAX_DESCRIPTION_LENGTH = 255;
```

Comment:
```suggestion
    private const MAX_DESCRIPTION_LENGTH = 255;
```

Reply:
As discussed will not done as used in the Tests